src/common/components/play/multiplayer/LobbyStatus.js

```diff
@@ -31,16 +19,48 @@ const LobbyStatus = (props) => {
     <span>Connecting ...</span>
   );
 
+  const renderPlayerName = (userData, clientId) => userData && !userData.uid.startsWith('guest_') ? 
+    renderRegisteredPlayer(userData, clientId) : renderGuestPlayerName(userData, clientId);
+
+  /* eslint-disable react/no-multi-comp */
+  const renderRegisteredPlayer = (userData, clientId) => (
+    <a 
+      style={{
+        fontStyle: clientId === myClientId ? 'italic' : 'normal',
+        textDecoration: 'underline',
+        color: '#666'
+      }}
+      href={userData && `/profile/${userData.uid}`}
+    >
+      {userData ? userData.displayName : clientId}
+    </a>
+  );
+
+  const renderGuestPlayerName = (userData, clientId) => (
+    <span 
+      style={{
+        fontStyle: clientId === myClientId ? 'italic' : 'normal',
+        color: '#666'
+      }}
+    >
+      {userData && userData ? userData.displayName : clientId}
+    </span>
+  );
+
   return (
     <Paper style={{padding: 20, marginBottom: 20}}>
       <div style={{position: 'relative'}}>
-        { connected ? <span className={classes.playersOnline}>
+        { connected ? <span>
             <b>{playersOnline.length} player(s) online: </b>
-            {playersOnline.map(clientId =>
-              <span key={clientId} style={{ fontStyle: clientId === myClientId ? 'italic' : 'normal' }}>
-                {userDataByClientId[clientId] ? userDataByClientId[clientId].displayName : clientId}
-              </span>
-            )}
+            {
+              playersOnline.map((clientId, idx) => 
+                <React.Fragment key={clientId}>
```

Comment:
nit: Can we use the `<>` shorthand for React fragments or do we not have the right babel plugins for that?

Reply:
I'm not sure 🤔